draft-ietf-quic-multipath.md

```diff
@@ -323,13 +323,27 @@ client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
 
-If validation succeed, the client can send non-probing, 1-RTT packets
-on the new paths.  In contrast with the specification in
+If validation succeed, the client can continue to use the path.
+If validation fails, the client MUST NOT use the path and can
+remove any status assiciated to the path initation attempt.
+{{Section 9.1 of QUIC-TRANSPORT}} introduces the concept of
+"probing" and "non-probing" frames. When the  multipath extension
+is negoatiated, the receiption of "non-probing"
+packet on a new path needs to be consider as an path initiation
+attempt that does not impact the path status of any existing
+path. Therefore, any frame can be send on a new path at any time
+as long as the congestion control limits are respected.
+
+Further, in contrast with the specification in
 {{Section 9 of QUIC-TRANSPORT}}, the server MUST NOT assume that
-receiving non-probing packets on a new path indicates an attempt
+receiving non-probing packets on a new path with a new Connection ID
+indicates an attempt
 to migrate to that path.  Instead, servers SHOULD consider new paths
 over which non-probing packets have been received as available
-for transmission.
+for transmission. Only the reception of "non-probing" frames on a new
```

Comment:
```suggestion
for transmission. Reception of QUIC packets on a new
```
I think it is considered migration when CID is the same, regardless of the types of the frames being conveyed?